Description: Fix reproducible build
 Use -ggdb2 instead of -ggdb3 for reproducible build
Author: Valentin Vidic <Valentin.Vidic@CARNet.hr>
Last-Update: 2017-10-26
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/configure.ac
+++ b/configure.ac
@@ -2087,7 +2087,7 @@
 fi
 
 if test "$GCC" = yes; then
-  CFLAGS="$CFLAGS -ggdb3"
+  CFLAGS="$CFLAGS -ggdb2"
   if
     cc_supports_flag -funsigned-char
   then
